File: C/arrays/choose_order_numbers.c
#include <stdio.h>

int main() {
	//Declaração
	int cod;
	float vetor[5];

	do {
		printf ("\nDigite o código referente a opção desejada: \n1 - Ordem Direta: \n2 - Ordem Inversa: \n0 - Encerrar\n");
		printf ("Opção selecionada: ");
		fflush (stdout);
		scanf ("%d", &cod);

		//Entrada
		if (cod == 1) {
			printf ("Digite 5 números reais:\n");
			fflush (stdout);
			for (int i = 0; i < 5; i++) {
				scanf ("\n%f", &vetor[i]);
			}
			//Saída
			printf ("Ordem direta: ");
			for (int i = 0; i < 5; i++) {
				printf ("%.1f ", vetor[i]);
			}
		}

		//Entrada
		if (cod == 2) {
			printf ("Digite 5 números reais:\n");
			fflush (stdout);
			for (int i = 0; i < 5; i++) {
			scanf ("\n%f", &vetor[i]);
			}

			//Saída
			printf ("Ordem inversa: ");
			for (int i = 4; i >= 0; i--) {
				printf ("%.1f ", vetor[i]);
			}
		}
	} while (cod != 0);
	printf ("\nPrograma finalizado com sucesso.");
}

File: Python/arrays/choose_order_numbers.py
# coding=utf8
# Declaração
vetor = []
codigo = 1

# Entrada/Processamento
while codigo != 0 and codigo <= 2:
    codigo = int (input("\nDigite o código numérico: \n1 - Ordem direta \n2 - Ordem inversa \n0 - Terminar o programa\n"))
    if codigo == 1:
        vetor.clear()
        for i in range(0, 5):
            num = float (input("Digite um número: "))
            vetor.append(num)
        #Saída
        for i in vetor:
            print (i, end = " ")
            
    if codigo == 2:
        vetor.clear()
        for i in range(0, 5):
            num = float (input("Digite um número: "))
            vetor.append(num)
        vetor.reverse()
        #Saída
        for i in vetor:
            print(i, end = " ")
print("\nPrograma finalizado com sucesso.")